kedro/templates/project/prompts.yml

```diff
@@ -1,3 +1,22 @@
+add_ons:
+  title: "Project Add-Ons"
+  text: |
+    Here you can select which add-ons you'd like to include.
+    Don't worry if you change your mind you can always add/remove these later.
```

Comment:
"Don't worry if you change your mind you can always add/remove these later.", we haven't really decided yet how to do this, so perhaps we should not add this in the description yet? cc: @yetudada @amandakys

Reply:
Opting for removing this for this iteration as this isn't implemented here.

Reply:
Yes, it should not be here. In this iteration, we will not allow our users to do this.